phpBB/develop/create_iconify_bundle.php

```diff
@@ -0,0 +1,277 @@
+<?php
```

Comment:
What do you think about moving this to a symfony command?
I see all the scripts in this folder as something to be removed in the future